unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/SectionController/MenuSections/SectionScenesController.cs

```diff
@@ -0,0 +1,173 @@
+﻿using DCL.Helpers;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+internal class SectionScenesController : SectionBase, IDeployedSceneListener, IProjectSceneListener
+{
+    internal const int MAX_CARDS = 3;
+    internal readonly SectionScenesView view;
+
+    private bool hasScenes = false;
+
+    public override ISectionSearchHandler searchHandler => hasScenes ? sceneSearchHandler : null;
+    public override SearchBarConfig searchBarConfig => new SearchBarConfig()
+    {
+        showFilterContributor = false,
+        showFilterOperator = false,
+        showFilterOwner = false,
+        showResultLabel = false
+    };
+
+    private readonly SceneSearchHandler sceneSearchHandler = new SceneSearchHandler();
+
+    internal Dictionary<string, SceneCardView> deployedViews;
+    internal Dictionary<string, SceneCardView> projectViews;
+    private List<SceneSearchInfo> searchList = new List<SceneSearchInfo>();
+
+    public SectionScenesController()
+    {
+        var prefab = Resources.Load<SectionScenesView>("BuilderProjectsPanelMenuSections/SectionScenesView");
+        view = Object.Instantiate(prefab);
+
+        view.btnProjectsViewAll.onClick.AddListener(()=> RequestOpenSection(SectionsController.SectionId.SCENES_PROJECT));
+        view.btnInWorldViewAll.onClick.AddListener(()=> RequestOpenSection(SectionsController.SectionId.SCENES_DEPLOYED));
```

Comment:
Shouldn't we unsubscribe these 2 subscriptions in some place (maybe in the `Dispose` function)?

Reply:
Dispose will Destroy view.gameObject and those two buttons are inside that GameObject so there is really no need to unsubscribe to those click events